packages/app-store/nextcloudtalk/zod.ts

```diff
@@ -1,10 +1,15 @@
 import { z } from "zod";
 
 export const appKeysSchema = z.object({
-  nextcloudTalkHost: z.string(),
-  nextcloudTalkPattern: z.string().optional(),
-  nextcloudTalkClientId: z.string(),
-  nextcloudTalkClientSecret: z.string(),
+  nextcloudTalkHost: z.string().describe("Your Nextcloud base URL"),
+  nextcloudTalkPattern: z
+    .string()
+    .optional()
+    .describe(
+      "Pattern according to which your created calls are named. You can use the following placeholders: {uuid}, {Title}, {Event Type Title}, {Scheduler}, {Organizer}, {Location} and {Team}"
+    ),
+  nextcloudTalkClientId: z.string().describe("Your Nextcloud OAuth Client's ID"),
+  nextcloudTalkClientSecret: z.string().describe("Your Nextcloud OAuth Client's secret"),
```

Comment:
I think it's understandable that it is the clientId and clientSecret fields from the props name

Reply:
Oh yes, agreed. I just still had them in there for testing. Removed them.